Mathlib/Geometry/Manifold/ContMDiff/Constructions.lean

```diff
@@ -1,17 +1,20 @@
 /-
 Copyright (c) 2020 Sébastien Gouëzel. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Sébastien Gouëzel, Floris van Doorn
+Authors: Sébastien Gouëzel, Floris van Doorn, Michael Rothgang
 -/
 import Mathlib.Geometry.Manifold.ContMDiff.Basic
 
 /-!
 ## Smoothness of standard maps associated to the product of manifolds
 
-This file contains results about smoothness of standard maps associated to products of manifolds
+This file contains results about smoothness of standard maps associated to products and sums
+(disjoint unions) of smooth manifolds:
 - if `f` and `g` are `C^n`, so is their point-wise product.
 - the component projections from a product of manifolds are smooth.
 - functions into a product (*pi type*) are `C^n` iff their components are
+- if `M` and `N` are `C^n` manifolds modelled over the same space, `Sum.inl` and `Sum.inr` are
```

Comment:
```suggestion
- if `M` and `N` are manifolds modelled over the same space, `Sum.inl` and `Sum.inr` are
```
I don't think you need any smoothness on `M` or `N` for this statement to be true.

Reply:
Indeed, corrected.